showcase/app/templates/page-components/accordion.gts

```diff
@@ -0,0 +1,583 @@
+              <:toggle>Item one</:toggle>
+              <:content>
+                <ShwPlaceholder @text="generic content" @height="40" />
+              </:content>
+            </HdsAccordionItem>
+          </SG.Item>
+        {{/each}}
+      </ShwGrid>
+
+      <ShwDivider @level={{2}} />
+
+      <ShwTextH4>isStatic</ShwTextH4>
+
+      <ShwGrid @columns={{2}} @gap="2rem" as |SG|>
+        {{#each TYPES as |type|}}
+          <SG.Item @label="isStatic=true">
+            <HdsAccordionItem @type={{type}} @isStatic={{true}}>
+              <:toggle>Item one</:toggle>
+              <:content>
+                <ShwPlaceholder @text="generic content" @height="40" />
+              </:content>
+            </HdsAccordionItem>
+          </SG.Item>
+
+          <SG.Item @label="isStatic=true, containsInteractive=true">
+            <HdsAccordionItem
+              @type={{type}}
+              @isStatic={{true}}
+              @containsInteractive={{true}}
+            >
+              <:toggle>Item one</:toggle>
+              <:content>
+                <ShwPlaceholder @text="generic content" @height="40" />
+              </:content>
+            </HdsAccordionItem>
+          </SG.Item>
+
+          <SG.Item @label="isStatic=true, isOpen=true">
+            <HdsAccordionItem
+              @type={{type}}
+              @isStatic={{true}}
+              @isOpen={{true}}
+            >
+              <:toggle>Item one</:toggle>
+              <:content>
+                <ShwPlaceholder @text="generic content" @height="40" />
+              </:content>
+            </HdsAccordionItem>
+          </SG.Item>
+
+          <SG.Item
+            @label="isStatic=true, containsInteractive=true, isOpen=true"
+          >
+            <HdsAccordionItem
+              @type={{type}}
+              @isStatic={{true}}
+              @containsInteractive={{true}}
+              @isOpen={{true}}
+            >
+              <:toggle>Item one</:toggle>
+              <:content>
+                <ShwPlaceholder @text="generic content" @height="40" />
+              </:content>
+            </HdsAccordionItem>
+          </SG.Item>
+        {{/each}}
+      </ShwGrid>
+
+      <ShwDivider @level={{2}} />
+
+      <ShwTextH4>Close</ShwTextH4>
+
+      <ShwFlex
+        {{style gap="2rem" marginBottom="2rem"}}
+        @direction="column"
+        as |SF|
+      >
+        <SF.Item @label="Close action within content">
+          <HdsAccordionItem>
+            <:toggle>Item one</:toggle>
+            <:content as |c|>
+              <button
+                type="button"
+                {{style padding=".25rem" marginBottom="1rem"}}
+                {{on "click" c.close}}
+              >Close</button>
+            </:content>
+          </HdsAccordionItem>
+        </SF.Item>
+      </ShwFlex>
+
+      <ShwDivider @level={{2}} />
+
+      <ShwTextH4>States</ShwTextH4>
+
+      <SectionAccordionItemStates />
```

Comment:
Not sure if this double format (code blocks with "mock" components / "section" components) is meant to show two possible directions, or if this is what is being proposed.

What I would suggest we to do is to stick to one format, in particular "everything is a section". Think of the format of how we include sub-sections in the `index` markdown files in the website. How this single format makes it simple to find things (_don't make me think™_) and has served us well, even when the file that was split contained just a few lines of text in some cases